src/openforms/registrations/contrib/stuf_zds/client.py

```diff
@@ -106,14 +120,16 @@ def _make_request(
         self,
         template_name: str,
         context: dict,
-        sync=False,
+        endpoint_type,
         soap_action: str = "",
     ) -> Tuple[Response, Element]:
 
         request_body = loader.render_to_string(template_name, context)
         request_data = self._wrap_soap_envelope(request_body)
 
-        url = self.service.get_endpoint(sync=sync)
+        url = self.service.get_endpoint(endpoint_type)
+
+        logger.debug("SOAP-request:\n{}\n{}".format(url, request_data))
```

Comment:
Let the logging framework do the formatting for you instead of interpolating this yourself. Any bugs in that will not cause the application to crash

```suggestion
        logger.debug("SOAP-request:\n%s\n%s", url, request_data)
```